src/main/scala/firrtl/Emitter.scala

```diff
@@ -190,6 +190,7 @@ class VerilogEmitter extends SeqTransform with Emitter {
 
   val outputSuffix = ".v"
   val tab = "  "
+  //scalastyle:off method.name
```

Comment:
I think if the PR uses `scalastyle:off`, then it would need to turn it back on at the end of the method?

There is an inline `scalastyle:ignore` that the PR could use on a per-line basis that might work better for the method name ones.

Reply:
> I think if the PR uses `scalastyle:off`, then it would need to turn it back on at the end of the method?

Thanks for catching that. The style in which I was using it to disable things for individual classes or methods is used in some other Chisel related code base, but as you pointed out, it actually stays off for the rest of the file.

I think in the end the thing to do is to close these PRs and add a reminder on the developer's meeting schedule to run `scalafmt` before releasing `1.4`.